src/server/ClientHandler.java

```diff
@@ -0,0 +1,126 @@
+package server;
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.Scanner;
+
+public class ClientHandler implements Runnable {
+    private int clientNum;
+    private String operator ;
+    private int result ;
+    private ServerSocket serverSocket = null;
+    private Socket connectionSocket = null;
+    private OutputStream output ;
+    private InputStream input ;
+
+    public ClientHandler(int clientNum, Socket connectionSocket, ServerSocket serverSocket) {
+        this.serverSocket = serverSocket;
+        this.connectionSocket = connectionSocket;
+        this.clientNum = clientNum;
+        operator = "";
+        result = 0;
+    }
+
+    @Override
+    public void run() {
```

Comment:
this function may be divided into smaller functions. its doing several functionalities